hod/options.py

```diff
@@ -64,4 +64,22 @@ def validate_pbs_option(options):
     if not options.workdir:
         _log.error('No workdir ("--workdir") provided')
         return False
+    if not options.hod_module:
+        _log.error('No hod-module ("--hod-module") provided')
+        return False
+
+    return True
+
+def validate_genconfig_option(options):
```

Comment:
rename to `validate_required_options` and call this from `validate_pbs_options`?

avoid duplicate code, it'll bite us in the ass later if we don't...